### ISSUE TITLE
Urinary ascites

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11316]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 37580961: **Urinary ascites** is an extremely rare condition that is correspondence with spontaneous bladder rupture. Symptoms of urinary ascites are vague and nonspecific, including acute lower abdominal pain, vomiting, nausea, dysuria, tenderness, and hematuria. 

PMID: 23190938 Neonatal ascites is an uncommon finding with a number of etiologies. Congenital **urinary ascites** is rare and it actually protects renal function by decompressing the urinary tract and preventing severe secondary changes to bladder function. Urinary ascites is commonly due to an obstructive uropathy or a neurogenic bladder. (...) We conclude that ruptured urachal diverticulum should be considered as a very rare cause of
urinary ascites

**Parent term (use hpo.jax.org/app)**
Ascites HP:0001541

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
urachal diverticulum ORPHA:431347

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding as
HP:6001378
Urinary ascites

Thanks!
